prov/tcp/src/tcpx_progress.c

```diff
@@ -666,6 +685,24 @@ void tcpx_progress_rx(struct tcpx_ep *ep)
 		tcpx_ep_disable(ep, 0);
 }
 
+void tcpx_progress_err(struct tcpx_ep *ep)
```

Comment:
I understand that ZEROCOPY support reports async completion in an error queue but I find the function name a bit confusing. I would suggest renaming it to something like `tcpx_progress_async`.

Reply:
I actually had it named that originally, then changed it to align with the POLL event IN->rx, OUT->tx, ERR->err.  It only handles zerocopy, so I could also go with progress_zerocopy, though I'm considering finding a shorter way of saying zerocopy (zcp? zerocp? bleh).